packages/plugin-avatar/src/avatar.js

```diff
@@ -0,0 +1,105 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import AvatarUrlBatcher from './avatar-url-batcher';
+import AvatarUrlStore from './avatar-url-store';
+import {SparkPlugin} from '@ciscospark/spark-core';
+import {defaults} from 'lodash';
+
+const Avatar = SparkPlugin.extend({
+  namespace: `Avatar`,
+
+  children: {
+    batcher: AvatarUrlBatcher
+  },
+
+  session: {
+    store: {
+      default() {
+        return new AvatarUrlStore();
+      },
+      type: `any`
+    }
+  },
+
+  /**
+   * @private
+   * Requests an avatar URL from the api
+   * @param {string} uuid
+   * @param {Object} options
+   * @param {integer} options.size
```

Comment:
I think the docs throughout this file should probably say `{number}` instead of integer